modules/REST/V1/Settings.php

```diff
@@ -103,10 +103,20 @@ public function update_item( $request ) {
 		);
 
 		foreach ( $options as $option_name => $new_value ) {
+			if ( $new_value === $this->options->get( $option_name ) ) {
+				continue;
+			}
```

Comment:
This should be done after sanitization, so boolans can be casted from `1` to `true` by the sanitization for example.

Reply:
Done in c16372da.